src/main/java/com/ipiecoles/java/java350/model/Employe.java

```diff
@@ -63,8 +63,9 @@ public Integer getNbRtt(LocalDate d){
         int var = 104;
         switch (LocalDate.of(d.getYear(),1,1).getDayOfWeek()){
             case THURSDAY: if(d.isLeapYear()) var =  var + 1; break;
-            case FRIDAY: if(d.isLeapYear()) var =  var + 2; else var =  var + 1;
+            case FRIDAY: if(d.isLeapYear()) var =  var + 2; else var =  var + 1; break;
```

Comment:
Pour respect du guide de style 
```suggestion
            case FRIDAY: 
                    if(d.isLeapYear()) {
                            var =  var + 2; 
                     } else {
                          var =  var + 1; 
                    }
                    break;
```